src/Input/Select/Select.tsx

```diff
@@ -61,6 +55,38 @@ class Select extends React.Component<Props, State> {
     document.addEventListener('click', this.handleOnBlur, false);
   }
 
+  componentDidUpdate(prevProps: Props) {
+    const hasDifferentChildren = !isEqual(this.props.children,  prevProps.children);
```

Comment:
This is just personal formatting preference but you can consider destructuring the props object to remove the need for `this.props`

`const { children, value, isLoading } = this.props`

Reply:
got it, updated